Client/src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+} from "@mui/material";
+import MySnackbar from "../Snackbar/MySnackbar";
+import { useDispatch, useSelector } from "react-redux";
+import { useState, useEffect } from "react";
+import { REQUEST_STATUSES } from "../../constants/apiRequestStatus";
+import {
+  loginFormVisibilityChanged,
+  addNewUser,
+  selectLoginError,
+  selectLoginStatus,
+  resetStatus,
+} from "../../slices/loginSlice";
+import "./Login.css";
+
+export default function Login(props) {
+  const handleCloseSnackBar = () =>
+    setSnackbarState((previous) => ({ ...previous, open: false }));
+
+  const [snackbarState, setSnackbarState] = React.useState({
+    open: false,
+    vertical: "top",
+    horizontal: "center",
+    severity: "success",
+    message: "",
+    handleClose: handleCloseSnackBar,
+  });
+  const [inputName, setInputName] = useState("");
+  const [inputPhoneNumber, setInputPhoneNumber] = useState("");
+
+  const dispatch = useDispatch();
+  const error = useSelector(selectLoginError);
+  const status = useSelector(selectLoginStatus);
+  const { vertical, horizontal, open, severity } = snackbarState;
```

Comment:
Seems you don't use destructed properties anywhere, do you need them?

Reply:
No, I forgot to remove them